src/main.py

```diff
@@ -180,6 +187,7 @@ def download(job_id):
 
         try:
             job = Job.fetch(job_id, connection=conn)
+            job_meta = job.get_meta()
```

Comment:
I can't seem to get the meta object when downloading, do you know how? @jgorset @Johannes-Andersen